pkg/instance/instance.go

```diff
@@ -959,6 +987,24 @@ func (i *Instance) SetPrometheusRemoteWriteExporter(endpoint string) error {
 	return nil
 }
 
+func (i *Instance) EnableTsharkCollector(volumeSize, s3AccessKey, s3SecretKey, s3Region, s3BucketName, s3KeyPrefix string) error {
```

Comment:
then enable can be updated to check if not nil to prevent multiple calls overwriting things by accident.